src/chrome/content/rules/BIBB.de.xml

```diff
@@ -0,0 +1,45 @@
+<!--
+	Federal Institute for Vocational Education and Training
+
+	Invalid certificate:
+		europass.bibb.de
+		exanda.bibb.de
+		good-practice.bibb.de
+		ldbb.bibb.de
+		www.ldbb.bibb.de
+		na.bibb.de
+		www.na.bibb.de
```

Comment:
Add returned certificates' Common Names.

Reply:
I've never seen any other ruleset where the certificates' common names are provided in the comment section. What's the usecase for this?

Reply:
Most likely for cases when domain is pointed to a CDN bucket using CNAME. See this ruleset for an example: https://github.com/EFForg/https-everywhere/blob/master/src/chrome/content/rules/Exmasters.com.xml. There's currently no rule that requires adding CNs to the comment, so I will approve this PR anyway.